libs/sdk-ui-ext/src/internal/utils/bucketHelper.ts

```diff
@@ -52,7 +53,7 @@ import {
     areObjRefsEqual,
 } from "@gooddata/sdk-model";
 
-export function isDateFilter(filter: IBucketFilter): filter is IDateFilter {
+export function isDateFilter(filter: IBucketFilter | IFiltersBucketItem): filter is IDateFilter {
```

Comment:
really this type is needed? It is strange

Reply:
Fixed. I added new type guard that is correct for this type